### DISCUSSION TITLE
Is it possible for me to change the name of a list?

### DISCUSSION THREAD

**Discussion by yanhao-pro** [community/community#128355] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Thank you for this awesome feature. I just wonder if I can change the name of a list. Maybe I can have typo in my initial input or just like to have another name for the list I created yesterday. Is it possible?

**Comment by davevad93** [comment, marked as answer]
Hi @yanhao-pro ,

Yes, you can change the name of a list, enter the list you want to edit, click on `Edit list`, type a new name and save it:

![1](https://github.com/community/community/assets/94308234/178f55d7-91ed-40ab-bd40-b239929d5f82)

![0](https://github.com/community/community/assets/94308234/0e18a195-2429-4567-bc61-a387dfb604c7)

**Reply by Maksymalnie** [reply]
Pytelmaksym@icloud.com

**Reply by Manialimani** [reply]
Thanks

**Reply by Hasannejad** [reply]
https://t.me/hamster_kombat_bOt/start?startapp=kentId6894963963

**Comment by Yooch3** [comment]
Help

**Reply by Maksymalnie** [reply]
Token

**Reply by Maksymalnie** [reply]
Bitkoin

**Reply by Necmeddinhammami** [reply]
N

**Reply by Yooch3** [reply]
> N

Hello

**Comment by eff24Alda** [comment]
Jeffrey Olivar Alda

**Comment by Hasannejad** [comment]
https://t.me/Hamster_kombat_bot/start?startapp=kentId6894963963

**Reply by Hasannejad** [reply]
https://t.me/Hamster_kombat_bot/start?startapp=kentId6894963963

**Reply by Hasannejad** [reply]
https://t.me/hamster_kombat_bOt/start?startapp=kentId6894963963

**Comment by Hasannejad** [comment]
> https://t.me/Hamster_kombat_bot/start?startapp=kentId6894963963

**Comment by Hasannejad** [comment]
> https://t.me/Hamster_kombat_bot/start?startapp=kentId6894963963

**Comment by Hasannejad** [comment]
https://t.me/hamster_kombat_bOt/start?startapp=kentId6894963963

**Comment by Hasannejad** [comment]
> https://t.me/hamster_kombat_bOt/start?startapp=kentId6894963963
